lib/rubygems/specification_policy.rb

```diff
@@ -224,6 +226,12 @@ def validate_dependencies # :nodoc:
     end
   end
 
+  def validate_required_ruby_version
+    if @specification.required_ruby_version.requirements == [Gem::Requirement::DefaultRequirement]
+      warning "make sure you specify the oldest ruby version that you want your gem to support by setting the `required_ruby_version` gemspec attribute"
```

Comment:
I wonder if this wording could lead to people setting `required_ruby_version = "<min_supported_version>"` instead of ``required_ruby_version = ">= <min_supported_version>"`? And that would actually prevent installation on any rubies, exception for the minimum supported one? 🤔

Should we clarify how it's supposed to be set?

Reply:
I have updated the message to make it clear.